docs/derivation-rules.saty

```diff
@@ -24,19 +24,19 @@ but extended by remote call, local call, list, etc.
     +small-block<
       +p{
         \BNFs[
-          (${e}, [${| v | x | fn | \{e, \cdots , e\} | \mathtt{let}\ x = e\ \mathtt{in}\ e | \mathtt{letrec}\ x = fn, \cdots , x = fn\ \mathtt{in}\ e |};
-                  ${| e\(e, \cdots , e\) | \mathtt{case}\ e\ \mathtt{of}\ pg \rightarrow e\; \cdots \;\ pg \rightarrow e\ \mathtt{end} | \mathtt{fun}\ f/a | \mathtt{\[}e\mathtt{\|}e\mathtt{\]} | \mathtt{\[\]} | \mathtt{fun}\ \mfa{m}{f}{a} \bnf-desc{term} |}]);
-          (${v}, [${| \mathtt{0} | \mathtt{\'ok\'} | \cdots \bnf-desc{constant} |}]);
+          (${e}, [${| v | x | fn | \{e, \cdots , e\} | \mathtt!{let}\ x = e\ \mathtt!{in}\ e | \mathtt!{letrec}\ x = fn, \cdots , x = fn\ \mathtt!{in}\ e |};
```

Comment:
`!` is added to `\mathtt` argument because `\mathtt` becomes to receive `inline-text` instead of `math`.